thunder/tests/test_nvfuser.py

```diff
@@ -850,3 +851,35 @@ def get_num_fusions(cfn):
     assert get_num_fusions(cfn_without_fusion) == 0
 
     nvfuserex.set_fuel(thunder.extend.FUEL_LEVEL.UNLIMITED)
+
+
+@instantiate(
+    dtypes=(thunder.float16, thunder.bfloat16), devicetypes=(devices.DeviceType.CUDA,), executors=(nvFuserExecutor,)
+)
```

Comment:
Is fp32, fp64 dtype supported?

Reply:
At the moment, we do have that restriction since we use fusedMultiplySum and MmaOp that only accept bf16/fp16. This restriction will also be lifted with the new IR nodes.